docs/en/writing-running-appium/touch-actions.md

```diff
@@ -13,6 +13,10 @@ These APIs allow you to build up arbitrary gestures with multiple actuators.
 Please see the Appium client docs for your language in order to find examples
 of using this API.
 
+**Note for W3C actions**
+
+[W3C actions](https://www.w3.org/TR/webdriver1/#actions) is also available in some drivers such as XCUITest, UIA2 and Espresso. [API doc](http://appium.io/docs/en/commands/interactions/actions/) and API docs of each binding help to understand how to call them.
```

Comment:
I would also mention, that the spec is mostly supported only partially (for example key actions are not supported by XCUITest) and with several platform-specific limitations (like 500ms delay for swipe)

Reply:
Do you have any good links to the limitation?
I could find some PRs/issues/comments in code tho.